Gemfile

```diff
@@ -26,6 +26,14 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 4.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'devise'
+gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
+gem 'pry'
+gem 'factory_bot_rails', '~> 5.0', '>= 5.0.2'
+gem 'hirb'  
+gem 'database_cleaner', '~> 1.7'
+gem 'rspec-rails', '~> 3.8', '>= 3.8.2'
```

Comment:
[Optional] Gems that you use for testing like `rspec-rails`,`pry`,`factory_bot_rails` is better to move to the `:test` group